docker/Dockerfile

```diff
@@ -3,7 +3,7 @@
 # DESCRIPTION: Amazon MWAA Local Dev Environment
 # BUILD: docker build --rm -t amazon/mwaa-local .
 
-FROM amazonlinux:2
+FROM amazonlinux:2.0.20230307.0
```

Comment:
@stories2 - specifying 2 pulls the latest version of amazonlinux 2, any reason you chose this specific version?